CalculatorTests/CalculatorBrainTests.swift

```diff
@@ -0,0 +1,289 @@
+    func testExampleSevenF() {
+        var brain = CalculatorBrain()
+        brain.setOperand(7)
+        brain.performOperation("+")
+        brain.setOperand(9)
+        brain.performOperation("√")
+        let description = brain.description!
+        let result = brain.result!
+        
+        XCTAssertEqual(description, "7+√(9)...")
+        XCTAssertEqual(result, 3)
+    }
+    
+    //Example found in task 7.g --- 7 + 9 √ = would show “7 + √(9) =“ (10 in the display)
+    func testExampleSevenG() {
+        var brain = CalculatorBrain()
+        brain.setOperand(7)
+        brain.performOperation("+")
+        brain.setOperand(9)
+        brain.performOperation("√")
+        brain.performOperation("=")
+        let description = brain.description!
+        let result = brain.result!
+        
+        XCTAssertEqual(description, "7+√(9)=")
+        XCTAssertEqual(result, 10)
+    }
+    
+    //Example found in task 7.h --- 7 + 9 = + 6 = + 3 = would show “7 + 9 + 6 + 3 =” (25 in the display)
+    func testExampleSevenH() {
+        var brain = CalculatorBrain()
+        brain.setOperand(7)
+        brain.performOperation("+")
+        brain.setOperand(9)
+        brain.performOperation("=")
+        brain.performOperation("+")
+        brain.setOperand(6)
+        brain.performOperation("=")
+        brain.performOperation("+")
+        brain.setOperand(3)
+        brain.performOperation("=")
+        let description = brain.description!
+        let result = brain.result!
+        
+        XCTAssertEqual(description, "7+9+6+3=")
+        XCTAssertEqual(result, 25)
+    }
+    
+    //Example found in task 7.i --- 7 + 9 = √ 6 + 3 = would show “6 + 3 =” (9 in the display)
+    func testExampleSevenI() {
+        var brain = CalculatorBrain()
+        brain.setOperand(7)
+        brain.performOperation("+")
+        brain.setOperand(9)
+        brain.performOperation("=")
+        brain.performOperation("√")
+        brain.setOperand(6)
+        brain.performOperation("+")
+        brain.setOperand(3)
+        brain.performOperation("=")
+        let description = brain.description!
+        let result = brain.result!
+        
+        XCTAssertEqual(description, "6+3=")
+        XCTAssertEqual(result, 9)
+    }
+    
+    //Example found in task 7.j --- 5 + 6 = 7 3 would show “5 + 6 =” (73 in the display)
+    func testExampleSevenJ() {
+        var brain = CalculatorBrain()
+        brain.setOperand(5)
+        brain.performOperation("+")
+        brain.setOperand(6)
+        brain.performOperation("=")
+        //Brain description is reset after this point but the label in the display is not ??
+        let description = brain.description!
+        brain.setOperand(73)
+        let result = brain.result!
+        
+        XCTAssertEqual(description, "5+6=")
+        XCTAssertEqual(result, 73)
+    }
+    
+    //Example found in task 7.k --- 4 × π = would show “4 x π =“ (12.566371 in the display)
+    func testExampleSevenK() {
```

Comment:
I think these are easier to understand than the initial ones where you delegate to the helper functions, because it's clear which operation(s) you're testing; if the tests' purpose is to test function X, then function X shouldn't be in abstracted away to a "helper". 

Are these tests like JUnit where members are re-initialised before each test is run? Perhaps you could keep the CalculatorBrain as a field in this test suite, and initialise in the setup method.